packages/cfd/src/Containers/mt5-compare-table-content.tsx

```diff
@@ -60,7 +60,7 @@ const Row = ({
     }
     if (is_platform_row && financial_restricted_countries) {
         values.financial_svg = { text: localize('MT5') };
-        values.financial_labuan = { text: localize('MT5') };
+        if (values.financial_labuan) values.financial_labuan = { text: localize('MT5') };
```

Comment:
@hamza-deriv, if there is a possibility of `values` being `undefined`, which I think is what you're trying to check for here, I suggest you use `values?.financial_labuan` instead 🤔